src/main/java/racingcar/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,55 @@
+package racingcar.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static int splitAndSum(String text) {
+
+        if (isBlank(text)) {
+            return 0;
```

Comment:
빈값일 때 왜 0을 반환하지? 
하는 생각을 할 수 있을 것 같아요
이 부분도 아래처럼 고쳐보면 좋겠습니다.

```suggestion
            return DEFAULT_VALUE;
```